draft/spec/index.html

```diff
@@ -281,9 +281,9 @@ <h2>OCFL Object</h2>
         specification.
     </p>
     <p>
-        The administrative information is specific to OCFL Objects. Implementers of OCFL are strongly encouraged to
-        store their own metadata serializations as files in an OCFL Object, but, in the context of the OCFL
-        Specification these are considered to be part of the contents of an OCFL Object.
+        An OCFL Object also has administrative information that identifies a folder as an OCFL Object, and also
+        provides a means of tracking changes to the contents of the object over time. Administrative files, such as
+        the <a>Inventory</a> files, are not considered to be part of the content of the object.
```

Comment:
Maybe it belongs in the "Implementation Notes", but this updated text loses the recommendation to consider client-provided metadata as "content".

Reply:
I think here it is OK to lose the comment about content including metadata here. It is in the bullet 1 just below